frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -555,7 +534,7 @@ export class HtmlDocument extends React.Component {
     const newText = e.target.value // because SyntheticEvent is pooled (react specificity)
     this.setState(prev => ({ content: { ...prev.content, raw_content: newText } }))
 
-    this.setLocalStorageItem('rawContent', newText)
+    setLocalStorageItem('rawContent', this.state.content, this.state.appName, newText)
```

Comment:
I think this line is missing `LOCAL_STORAGE_ITEM_TYPE.RAW_CONTENT`

Reply:
```suggestion
    setLocalStorageItem(LOCAL_STORAGE_ITEM_TYPE.RAW_CONTENT, this.state.content, this.state.appName, newText)
```